src/components/Link.js

```diff
@@ -22,7 +22,7 @@ const NextComposed = React.forwardRef((props, ref) => {
   return (
     <NextLink href={href} as={as}>
       <a
-        style={{ display: 'block' }}
+        style={{ display: 'flex' }}
```

Comment:
Could you change this to `inline-flex`? We use this Link component in a lot of places so we do not want any layout changes caused by `flex`.

Reply:
as mentioned in the NOTE section in the PR description, this should not affect any layout changes as we are wrapping only one component inside Link at a time

Reply:
changes updated